policyengine_us/variables/gov/states/ca/tax/income/alternative_minimum_tax/amti/ca_itemized_ded_limitation.py

```diff
@@ -0,0 +1,34 @@
+from policyengine_us.model_api import *
+
+
+class ca_itemized_ded_limitation(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "California itemized deductions limitation"
+    defined_for = StateCode.CA
+    unit = USD
+    definition_period = YEAR
+    reference = "https://www.ftb.ca.gov/forms/2022/2022-540-p.pdf"
+
+    def formula(tax_unit, period, parameters):
+        filing_status = tax_unit("filing_status", period)
+        p = parameters(
+            period
+        ).gov.states.ca.tax.income.deductions.itemized.limit
+        agi = tax_unit("ca_agi", period)
+        itemized_ded = tax_unit("ca_itemized_deductions", period)
+        agi_with_itemized_deductions = agi + itemized_ded
+
+        itemized_ded_over_limitation = where(
```

Comment:
Not sure if this captures the logic accurately but please break up the logic into steps like these throughout
```suggestion
        agi_threshold = p.agi_threshold[filing_status]
        limitation_applies = agi_with_itemized_deductions > agi_threshold
        itemized_ded_over_limitation = limitation_applies * itemized_ded
```